src/drop-down-menu.jsx

```diff
@@ -211,17 +212,19 @@ const DropDownMenu = React.createClass({
         }
       }
     }
-
     let selectedItem = this.props.menuItems[selectedIndex];
     if (selectedItem) {
-      displayValue = selectedItem[labelMember];
+      displayValue = selectedItem[labelMember] || selectedItem[displayMember];
```

Comment:
I see what you mean now (https://github.com/callemall/material-ui/pull/2285#issuecomment-162627521)